osu.Game/Screens/Play/HUD/PlayerAvatar.cs

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Allocation;
+using osu.Framework.Bindables;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Game.Configuration;
+using osu.Game.Localisation.SkinComponents;
+using osu.Game.Skinning;
+using osu.Game.Users.Drawables;
+using osuTK;
+
+namespace osu.Game.Screens.Play.HUD
+{
+    public partial class PlayerAvatar : CompositeDrawable, ISerialisableDrawable
+    {
+        [SettingSource(typeof(SkinnableComponentStrings), nameof(SkinnableComponentStrings.CornerRadius), nameof(SkinnableComponentStrings.CornerRadiusDescription))]
+        public new BindableFloat CornerRadius { get; set; } = new BindableFloat
+        {
+            MinValue = 0,
+            MaxValue = 63,
```

Comment:
I'm not a reviewer, but `MaxValue` shouldn't be 64?
If the size of the `PlayerAvatar` is 128 then the `MaxValue` of the corner radius should be half of it
(I tried it locally and it's not completely round when dragging the slider to the max, it's 2 pixels off)